tests/unit/helpers.py

```diff
@@ -34,3 +36,10 @@ def get_testdata_file(file_name: str) -> str:
     file_path = get_testdata_file_path(file_name)
     with open(file_path) as fp:
         return fp.read()
+
+def get_testdata(file_name: str) -> Dict:
+    config = get_testdata_file(file_name)
+    if file_name[-4] == "json":
+        return json.loads(config)
+    else:
+        return yaml.safe_load(config)
```

Comment:
We can use this one for the YAML file parsing (should be safer): https://github.com/pre-commit/pre-commit/blob/9eccd9c35b9fada1285eda975f8dd033635b250b/pre_commit/util.py#L23-L24

Reply:
Done